examples/BarChart.elm

```diff
@@ -0,0 +1,27 @@
+module OrdinalBands where
+
+import Svg exposing (Svg)
+import Plot exposing (..)
+import Plot.Scale as Scale
+import Plot.Axis as Axis
+
+main : Svg
+main =
+  let
+    xScale = Scale.ordinalBands ["a", "b", "c", "d"] (0, 800) 0.2 0.2
+    yScale = Scale.linear (0, 400) (800, 0) 10
```

Comment:
Will it be more confusing to new people not use to (0, 0) being the top left or more confusing to people use to this fact and making the origin at the bottom left instead? Both vega and D3 are consistent with the origin being the top left which requires your y values to be "flipped". 

I think for consistency it is better to accept the origin being the top left. Otherwise you would have to note that a range is the interval in pixels that you would like your inputs mapped to except for y values, which are reversed.